src/components/temperature-scale.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from "react";
+import Draggable from "react-draggable";
+import DragIcon from "../assets/drag.svg";
+import * as css from "./temperature-scale.scss";
+import { temperatureScale } from "../temperature-scale";
+
+interface IProps {}
+interface IState {
+  expanded: boolean;
+}
+
+const getFahrenheit = (celcius: number) => {
```

Comment:
"celcius" => "celsius"

Reply:
And there was me making double-sure to get the spelling of Fahrenheit correct! Good spot :)